package.json

```diff
@@ -44,7 +44,7 @@
     "eslint": "^5.13.0",
     "eslint-plugin-babel": "^5.3.0",
     "mocha": "^5.2.0",
-    "nyc": "^13.2.0",
+    "nyc": "^13.3.0",
```

Comment:
Please don't include unrelated changes in a single pull request. Since this change isn't related to the purpose of this PR, it should be done in a separate PR.